src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -603,14 +569,11 @@ const todo = await API.graphql({
 </Block>
 </BlockSwitcher>
 
-[comment]: # (EXPANDER end)
-  
-### Cancel mutation requests
+</Accordion>
 
-[comment]: # (Brief explanation on how canceling mutations works, what happens in the backend, and how to run the command – taken from https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/graphqlapi/js/cancel-request.mdx)
+### Cancel mutation requests
 
 You may cancel any query or mutation request made through the API category of Amplify by keeping a reference to the promise returned.
```

Comment:
Q from Jim during his initial edit pass - Will this sentence be clear to all readers? Can we spell this out in more detail?

Reply:
```suggestion
You can cancel any GraphQL API request calling `.cancel` on the GraphQL request promise that's returned by `API.graphql(...)`.
```